.buildbot.sh

```diff
@@ -1,7 +1,15 @@
 #!/bin/sh
```

Comment:
This should be `bash` like [here](https://github.com/capablevms/llvm-function-split/blob/main/.buildbot.sh). Do not use `cheri-examples` as base but `llvm-funciton-extract` since I am using `riscv64-purecap`. 
I have a syntax error when trying to test this into the running container. We are using *arrays* here so it is not standard POSIX :)